docs/sanic/logging.md

```diff
@@ -14,7 +14,7 @@ from sanic.config import LOGGING
 # The default logging handlers are ['accessStream', 'errorStream']
 # but we change it to use other handlers here for demo purpose
 LOGGING['loggers']['network']['handlers'] = [
-    'accessTimedRotatingFile', 'errorTimedRotationgFile']
+    'accessTimedRotatingFile', 'errorTimedRotatingFile']
```

Comment:
Can you change these two to `accessSyslog` and `errorSyslog`?

The file loggers were removed in #706